guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,14 +4,45 @@
 [id="katello-headline-features"]
 == Headline Features
 
-There are no highlights with Katello {KatelloVersion}.
+=== Entitlement Mode Removed in Favor of Simple Content Access
+Simple Content Access (SCA) is now the only content access mode supported.
+When upgrading Katello, all existing organizations that use entitlement mode will migrate automatically to SCA.
+This change is not reversible.
+
+Toggling SCA from the Organization Edit page, the Organization Create page, and the Organization List page is no longer available.
+
+=== Removed System Purpose-Related Host Statuses
+The following host statuses are removed:
+* Subscription Status (this column has also been removed from the All Hosts table)
+* System Purpose Status
+* SLA Status
+* Role Status
+* Usage Status
+* Addon Status
```

Comment:
Was it spelled like this in the UI? Not spelled Add-on or Add on?

Reply:
Unfortunately yes :confounded: 

![image](https://github.com/theforeman/foreman-documentation/assets/22042343/0230ce44-83fd-47f7-8b7f-df6020bad3af)

Reply:
Addons status sounds icky to me. Just like fathers in law. I think it's ok as is?

Reply:
https://www.theonion.com/william-safire-orders-two-whoppers-junior-1819565735